manimlib/mobject/probability.py

```diff
@@ -187,15 +186,14 @@ def add_axes(self):
         y_axis.add(ticks)
 
         if self.include_x_ticks == True:
-            ticks1 = VGroup()
             widths = np.linspace(0, self.width, self.n_ticks_x + 1)
             label_values = np.linspace(0, len(self.bar_names), self.n_ticks_x + 1)
             for x, value in zip(widths, label_values):
-                tick1 = Line(UP*0.05, DOWN*0.05)
-                tick1.set_width(self.tick_width_x)
+                tick1 = Line(UP, DOWN)
+                tick1.set_height(self.tick_height)
                 tick1.move_to(x * RIGHT)
-                ticks.add(tick1)
-            x_axis.add(ticks1)
+                ticks.add(tick)
+            x_axis.add(ticks)
```

Comment:
Oh, I think you need a new VGroup for `x_ticks` and add it to `x_axis`, instead of still using `ticks`.

Reply:
Should I also change the ticks variables names on y-axis so that the names on x and y-axis have the same format?

Reply:
that's better